src/MyEventEmitter.js

```diff
@@ -1,14 +1,86 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, listener) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+    this.events[eventName].push(listener);
+  }
+
+  // once(eventName, listener) {
+  //   if (!this.events[eventName]) {
+  //     this.events[eventName] = [];
+  //   }
+  //   this.events[eventName].push({ cb: listener, onetime: true });
+  // }
```

Comment:
Remove all comments and commented code.
Don't leave commented code at all, it's a bad practice